src/Game/Systems/CapturePointArrowHUDSystem.cpp

```diff
@@ -0,0 +1,137 @@
+                    } else if (CPID == 4) {
+                        //Home point for one team
+                        home2 = EntityWrapper(m_World, cCP.EntityID);
+                    }
+                }
+                //Check what team is the owner of Home1 and set their target to the next capturepoint
+                if(!home1.Valid() || !home2.Valid()) {
+                    return;
+                }
+                if((int)home1["CapturePoint"]["HomePointForTeam"] == redTeam) {
+                    m_RedTeamCurrentTarget = target1;
+                } else if ((int)home1["CapturePoint"]["HomePointForTeam"] == blueTeam) {
+                    m_BlueTeamCurrentTarget = target1;
+                }
+
+                //Check what team is the owner of Home2 and set their target to the next capturepoint
+                if ((int)home2["CapturePoint"]["HomePointForTeam"] == redTeam) {
+                    m_RedTeamCurrentTarget = target2;
+                } else if ((int)home2["CapturePoint"]["HomePointForTeam"] == blueTeam) {
+                    m_BlueTeamCurrentTarget = target2;
+                }
+            }
+        }
+        //if red team, get red team next point, otherwise blue team next point.
+        //Untill this is awailable we will just use the hardcoded value in the component.
+        //This will also give us a position, so we wont need to loop through all capturePoints.
+        glm::vec3 pos;
+        if(currentTeam == redTeam) {
+            pos = m_RedTeamCurrentTarget;
+        } else if (currentTeam == blueTeam) {
+            pos = m_BlueTeamCurrentTarget;
+        }
+
+        pos = currentTeam == redTeam ? m_RedTeamCurrentTarget : currentTeam == blueTeam ? m_BlueTeamCurrentTarget : glm::vec3(0.f);
+
+        glm::vec3& arrowOri = ArrowEntity["Transform"]["Orientation"];
+        glm::vec3 lookVector = glm::normalize(Transform::AbsolutePosition(ArrowEntity) - pos); //Maybe should be player instead
+        float pitch = std::asin(-lookVector.y);
+        float yaw = std::atan2(lookVector.x, lookVector.z);
+        arrowOri.x = pitch;
+        arrowOri.y = yaw;
+        arrowOri.z = 0.f;
+        EntityWrapper parent = ArrowEntity.Parent();
+        if (parent.Valid()) {
+            arrowOri -= Transform::AbsoluteOrientationEuler(parent);
+        }
+    }
+}
+
+bool CapturePointArrowHUDSystem::OnCapturePointCaptured(Events::Captured& e)
+{
+    if (!e.NextCapturePoint.HasComponent("Team"))
+    {
+        return 0;
+    }
+
+    auto cTeam = e.NextCapturePoint["Team"];
+    
+    int redTeam = (int)cTeam["Team"].Enum("Red");
+    int blueTeam = (int)cTeam["Team"].Enum("Blue");
+    int spectatorTeam = (int)cTeam["Team"].Enum("Spectator");
+    int target = -1;
+
+    if (e.NextCapturePoint.HasComponent("CapturePoint")) {
+        target = (int)e.NextCapturePoint["CapturePoint"]["CapturePointNumber"];
+    } else {
+        return 0;
+    }
+
+    if(e.TeamNumberThatCapturedCapturePoint == redTeam) {
```

Comment:
What will happen when an enemy takes your capture point? Does the target change to a previousCapturePoint?